pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -355,18 +373,16 @@ const MemoryRow = ({ memorySize, memorySizeUnit, nodeMaxMemory, onValueChanged,
                     initialUnit={memorySizeUnit}
                     onValueChange={e => onValueChanged('memorySize', e.target.value)}
                     onUnitChange={value => onValueChanged('memorySizeUnit', value)} />
-                {validationStateMemory === "error" &&
-                <HelpBlock>
-                    <p className="text-danger">{validationFailed.memory}</p>
-                </HelpBlock> }
-                {nodeMaxMemory &&
                 <HelpBlock id="memory-size-helpblock">
-                    {cockpit.format(
-                        _("Up to $0 $1 available on the host"),
-                        Math.floor(convertToUnit(nodeMaxMemory, units.KiB, memorySizeUnit)),
-                        memorySizeUnit,
-                    )}
-                </HelpBlock>}
+                    <p>{validationStateMemory === "error" && validationFailed.memory}</p>
+                    <p>{nodeMaxMemory &&
+                            cockpit.format(
+                                _("Up to $0 $1 available on the host"),
+                                Math.floor(convertToUnit(nodeMaxMemory, units.KiB, memorySizeUnit)),
+                                memorySizeUnit,
+                            )}</p>
+                    <p>{recommendedMemoryHelpBlock && recommendedMemoryHelpBlock}</p>
```

Comment:
I believe this will work the same even if you will have only `<p>{recommendedMemoryHelpBlock}</p>`. No need to `recommendedMemoryHelpBlock` variable twice.

Reply:
Ups,  not really, I wanted to put the condition outiside of the <p>. Same for the above. This needs to be fixed. I will repost.

Reply:
It's needed. See fix.